src/helpers/topicPropsBuilder.js

```diff
@@ -126,20 +126,25 @@ function topicPropsBuilder(regEntry, sourceEntry, config) {
 
   // building src and dest locations for release notes
   const releaseNotes = config.docu.folders.releaseNotes;
-  const rnBaseLocation = isWithoutReleaseNotes ? false : `${basicSrcLocation}/${releaseNotes}`;
+  const rnBaseLocation = hasReleaseNotes ? `${basicSrcLocation}/${releaseNotes}` : false;
   const rnSrcLocation = (rnBaseLocation) ? `${rnBaseLocation}/**` : false;
-  const rnBaseLocationInternal = isWithoutReleaseNotes ? false : `${basicSrcLocation}/internal/${releaseNotes}`;
+  const rnBaseLocationInternal = hasReleaseNotes ? `${basicSrcLocation}/internal/${releaseNotes}` : false;
   const rnSrcLocationInternal = (rnBaseLocationInternal) ? `${rnBaseLocationInternal}/**` : false;
-  const rnDestLocation = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/rn/${type}/${shortName}/latest`;
-  const rnDestLocationWithoutVersion = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/rn/${type}/${shortName}`;
-  const rnDestLocationInternal = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}/latest`;
-  const rnDestLocationInternalWithoutVersion = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}`;
+  const rnDestLocation = hasReleaseNotes ? `${config.skeletonSrcDestination}/rn/${type}/${shortName}/latest` : false;
+  const rnDestLocationWithoutVersion = hasReleaseNotes ? `${config.skeletonSrcDestination}/rn/${type}/${shortName}` : false;
+  const rnDestLocationInternal = hasReleaseNotes ? `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}/latest` : false;
+  const rnDestLocationInternalWithoutVersion = hasReleaseNotes ? `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}` : false;
 
   /*
   adding base uri - service proxy url
   */
 
-  const defaultBaseUriDomain = config.defaultBaseUriDomain || '';
+  if(!sourceEntry.baseUri && !config.defaultBaseUriDomain){
+    throw new Error(`Base URI register ${name} is not set.
```

Comment:
Base URI for service ${name} is not set.
Please set base URI value or provide default in your config file.